appserver/admingui/cluster/src/main/resources/dg/dgGeneral.jsf

```diff
@@ -0,0 +1,210 @@
+                    />
+                </sun:button>
+            </sun:panelGroup>
+        </facet>
+    </sun:title>
+
+    <sun:button id="startButton" text="$resource{i18ncs.button.startDG}" disabled="#{disableStart}" primary="#{false}"
+        onClick="if ( getConfirm(this,'$resource{i18ncs.msg.JS.confirmStartDGs}') )
+                { return submitAndDisable(this, '$resource{i18n.button.Processing}');}  else {return false;} " >
+        <!command
+            println("Starting Deployment Group...");
+            println("#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/start-deployment-group");
+            gf.restRequest(endpoint="#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/start-deployment-group" method="POST");
+            gf.redirect(page="#{pageSession.selfPage}&alertType=${alertType}&alertSummary=${alertSummary}&alertDetail=${alertDetail}");
+        />
+    </sun:button>
+
+    <sun:button id="stopButton" text="$resource{i18ncs.button.stopDG}" disabled="#{disableStop}" primary="#{false}"
+        onClick="if ( getConfirm(this,'$resource{i18ncs.msg.JS.confirmStopDGs}') )
+                { return submitAndDisable(this, '$resource{i18n.button.Processing}');}  else {return false;} " >
+        <!command
+            println("Stopping Deployment Group...");
+            println("#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/stop-deployment-group");
+            gf.restRequest(endpoint="#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/stop-deployment-group" method="POST");
+            gf.redirect(page="#{pageSession.selfPage}&alertType=${alertType}&alertSummary=${alertSummary}&alertDetail=${alertDetail}");
+        />
+    </sun:button>
+
+    <sun:button id="restartButton" text="$resource{i18ncs.button.restartDG}" disabled="#{disableStop}" primary="#{false}"
+        onClick="if ( getConfirm(this,'$resource{i18ncs.msg.JS.confirmRestartDGs}') )
+                { return submitAndDisable(this, '$resource{i18n.button.Processing}');}  else {return false;} " >
+        <!command
+            println("Restarting Deployment Group...");
+            println("#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/restart-deployment-group");
+            gf.restRequest(endpoint="#{sessionScope.REST_URL}/deployment-groups/deployment-group/#{pageSession.encodedDGName}/restart-deployment-group" method="POST");
+            gf.redirect(page="#{pageSession.selfPage}&alertType=${alertType}&alertSummary=${alertSummary}&alertDetail=${alertDetail}");
+        />
+    </sun:button>
+
+    <event>
+        <!afterCreate
+            getUIComponent(clientId="#{sheetId}"  component=>$attribute{component})
+            includeIntegrations(type="org.glassfish.admingui:ejbTimerButtons" root="#{component}");
```

Comment:
This button breaks, giving a failure:

RestResponse.getResponse() gives FAILURE.  endpoint = 'http://localhost:4848/management/domain/list-timers?target='; attrs = '{}']]

Once broken it redirects to the cluster -> EJB Timer page, so clicking on application takes you to cluster -> applications rather than deployment group -> applications

Reply:
yeah that will be addressed in the EJB Timers JIRA which is next ;-)